core/assets/bundles/bundle_de.properties

```diff
@@ -209,31 +209,31 @@ save.playtime = Spielzeit: {0}
 warning = Warnung.
 confirm = Bestätigen
 delete = Löschen
-view.workshop = View In Workshop
-workshop.listing = Edit Workshop Listing
+view.workshop = In der Werkstatt anschauen
```

Comment:
Im Workshop ansehen

Reply:
OK

Reply:
Why shouldn't we translate the English word workshop into German?

Reply:
Because it's not translated in the german steam translation also. It's super confusing, also it's more like a noun.